test/integration/collection-management/view.test.ts

```diff
@@ -7,9 +7,8 @@ describe('Views', function () {
   let db: Db;
 
   beforeEach(async function () {
-    const configuration = this.configuration;
     client = this.configuration.newClient();
-    db = client.db(configuration.db);
+    db = client.db('views');
```

Comment:
impacting other tests after changing `setupDatabase`, using a new name avoids the issue.

Reply:
Why did changing setupDatabase break this + other tests?  If setupDatabase is correct, it shouldn't matter what other tests do?